### ISSUE TITLE
Unable to use <space> and navigate with arrow key inside the text input of `<paper-input>` inside `<my-view1>`

### ISSUE THREAD

**Issue by andrewspy** [PolymerElements/polymer-starter-kit#937]
### Description
I can't seem to navigate the text input with arrow key or enter `<space>` when using `<paper-input>` element inside `<my-view1>`

### Expected outcome

Able to use `<space>` and navigate with arrow key inside the text input of `<paper-input>`.

### Actual outcome

Unable to use `<space>` and navigate with arrow key inside the text input of `<paper-input>`.

### Live Demo

N/A

### Steps to reproduce

1. Install `paper-input` element with `bower`.
2. Put a `paper-input` element in my-view.html as below.
3. Open the page in a web browser.

```html
<link rel="import" href="../bower_components/polymer/polymer.html">
<link rel="import" href="shared-styles.html">

<dom-module id="my-view1">
  <template>
    <style include="shared-styles">
      :host {
        display: block;

        padding: 10px;
      }
    </style>

    <paper-input></paper-input>
    
    <div class="card">
      <div class="circle">1</div>
      <h1>View One</h1>
      <p>Ut labores minimum atomorum pro. Laudem tibique ut has.
      <p>Lorem ipsum dolor sit amet, per in nusquam nominavi periculis, sit elit oportere ea.Lorem ipsum dolor sit amet, per in nusquam nominavi periculis, sit elit oportere ea.Cu mei vide viris gloriatur, at populo eripuit sit.</p>
    </div>
  </template>

  <script>
    Polymer({
      is: 'my-view1'
    });
  </script>
</dom-module>
```

### Browsers Affected

- [x] Chrome
- [ ] Firefox
- [ ] Safari 9
- [ ] Safari 8
- [ ] Safari 7
- [ ] Edge
- [ ] IE 11
- [ ] IE 10

**Comment by robdodson** [comment]
You forgot to import the paper-input in your code snippet.

I'm not sure what you mean when you say you can't "navigate the text input with arrow key or enter <space>". To focus the paper-input you'll want to press `Tab`.

**Comment by andrewspy** [comment]
@robdodson I did import `paper-input` element inside `my-app.html`, and I can actually "see" the `<paper-input>` element.

What I meant for "unable to navigate" is that when the text input is focus, I can type things inside, except I can't move the text cursor with arrow keys, or type a `<space>` character inside. Seems like `iron-behaviors` is blocking the `<space>` and arrow keys when the text input is focus. (I am not familiar with internal of `iron-behaviors`).

Being said that, I have tested the same code on Firefox and Edge, and both are working as expected. Problem only happened on Chrome.
